docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -197,7 +197,7 @@
       block:
       - name: Find deployment-manager app helm-chart in application directory
         find:
-          paths: /usr/local/share/applications/helm/
+          paths: "{{ manager_app_chart | dirname }}"
           patterns: 'deployment-manager*.tgz'
```

Comment:
I'm fine with this change for now as the expectation of the tarball name is like this pattern. But keep in mind, if we rename the tarball name and give it to the overrides, this pattern may not fit therefore the failure will be caused by it.

In these cases, we may need to think tasks to check if there's a wildcard in the file name, and process it differently